internal/usage/usage.go

```diff
@@ -383,22 +383,23 @@ dbName and collection are required only for built-in roles.`
-	IssuerURI             = "Unique string that identifies the issuer of the OIDC metadata/discovery document URL."
-	AssociatedDomains     = "List of domains associated with the Identity Provider."
-	RequestedScopes       = "Scopes that MongoDB applications will request from the authorization endpoint.	"
-	Protocol              = "Protocol used to authenticate the user. Valid value is OIDC or SAML."
-	IdentityProviderID    = "Unique identifier of the identity provider."
+	RetentionUnit              = "Unit of time in which Atlas measures snapshot retention: 'days' 'weeks' 'months'. "
+	RetentionValue             = "Duration in days, weeks, or months that Atlas retains the snapshot. For less frequent policy items, Atlas requires that you specify a value greater than or equal to the value specified for more frequent policy items."
+	ProjectConfigFilename      = "Path to the JSON configuration file that defines project configuration settings. To learn more about project configuration files for the Atlas CLI, see https://dochub.mongodb.org/core/project-config-file."
+	FederationSettingsID       = "Unique identifier of the federation settings."
+	OIDCType                   = "OIDC method by which the provided database user is authenticated. Valid values are NONE, USER, or IDP_GROUP. If you set this to USER or GROUP_ID, the user authenticates with OIDC."
+	IdpType                    = "Type of Identity Provider. Valid values are WORKFORCE or WORKLOAD."
+	Audience                   = "Identifier of the intended recipient of the token."
+	AuthorizationType          = "Type of authorization. Valid values are NONE, JWT, or SAML."
+	ClientID                   = "Client identifier that is assigned to an application by the Identity Provider.	"
+	Description                = "Description of the Identity Provider."
+	DisplayName                = "Display name of the Identity Provider."
+	GroupsClaim                = "Identifier of the claim which contains IdP Group IDs in the token."
+	UserClaim                  = "Identifier of the claim which contains the user ID in the token."
+	IssuerURI                  = "Unique string that identifies the issuer of the OIDC metadata/discovery document URL."
+	AssociatedDomains          = "List of domains associated with the Identity Provider."
+	RequestedScopes            = "Scopes that MongoDB applications will request from the authorization endpoint.	"
+	Protocol                   = "Protocol used to authenticate the user. Valid value is OIDC or SAML."
+	IdentityProviderID         = "Unique identifier of the identity provider."
+	ConnectedOrgConfigFilename = "Path to a JSON configuration file that defines connected orgs configurations. To learn more about connected org configuration file format, see the request body in https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Federated-Authentication/operation/updateConnectedOrgConfig."
```

Comment:
can be addressed in a follow-up PR: can you please ask docs for a dochub URL for the json config file. An example can be found here https://github.com/mongodb/mongodb-atlas-cli/blob/e7398653491c1a406819db58eba66483613f3c33/internal/usage/usage.go#L388